DSharpPlus/Entities/DiscordWidgetSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+	/// <summary>
+	/// Represents a Discord guild's widget settings.
+	/// </summary>
+	public class DiscordWidgetSettings
+    {
+        internal DiscordGuild Guild { get; set; }
+
+        [JsonProperty("channel_id", NullValueHandling = NullValueHandling.Ignore)]
+        internal ulong ChannelId { get; set; }
```

Comment:
Imo, you should expose the channel id in case the channel is not in cache, to at least know the id.